portal/src/main/webapp/js/lib/igv.js

```diff
@@ -5122,10 +5122,41 @@ var igv = (function (igv) {
             }
             return;
         }
+        var div = $("#igvDiv")[0],
+            options = {
+                showNavigation: true,
+                showRuler: true,
+                genome: "hg19",
+                locus: feature,
+                tracks: [
+                    {
+                        url: "http://localhost:8080/cbioportal/api/copynumbersegments",
```

Comment:
Good catch! I just updated the url link by removing localhost.
@sheridancbio  just brought up a very good point that we might need to update their liscence since we modified their library. I will update it once we find out how to do it.